public/app/features/provisioning/hooks/useCreateOrUpdateRepository.ts

```diff
@@ -13,14 +13,17 @@ export function useCreateOrUpdateRepository(name?: string) {
   const [testConfig, testRequest] = useCreateRepositoryTestMutation();
 
   const updateOrCreate = useCallback(
-    async (data: RepositorySpec) => {
+    async (data: RepositorySpec, token?: string) => {
+      const secure = token?.length ? { token: { create: token } } : undefined;
```

Comment:
Are secure values sent back to FE? I'm thinking about the case when editing config and sending data back to BE, considering we're using PUT for that, will missing token field be an issue?